frame/session/src/lib.rs

```diff
@@ -568,16 +547,86 @@ decl_module! {
 			}
 		}
 	}
+
+	/// Error for the session pallet.
+	#[pallet::error]
+	pub enum Error<T> {
+		/// Invalid ownership proof.
+		InvalidProof,
+		/// No associated validator ID for account.
+		NoAssociatedValidatorId,
+		/// Registered duplicate key.
+		DuplicatedKey,
+		/// No keys are associated with this account.
+		NoKeys,
+		/// Key setting account is not live, so it's impossible to associate keys.
+		NoAccount,
+	}
+
+	#[pallet::call]
+    impl<T: Config> Pallet<T> {
```

Comment:
Replace space with tabs:

```suggestion
	impl<T: Config> Pallet<T> {
```

Reply:
The lines below this line needs to be changed to using tabs as well.